docs: add virtual hosts articles ADD and EDIT (#945)
devopness/devopness: docs/virtual-hosts/add-virtual-host.md

---
title: Add a Virtual host

> intro: Add a Virtual host to define how your applications accessible on the internet by IP address or domain name

Reviewer: ```suggestion
intro: Add a Virtual Host to make an application accessible on the internet, by server IP address or a domain name
```
